iso_msgs/msg/CoordinateSystem.msg

```diff
@@ -0,0 +1,8 @@
+uint32 COORDINATE_SYSTEM_ETRS89 = 0
+uint32 COORDINATE_SYSTEM_NAD83 = 1
+uint32 COORDINATE_SYSTEM_ITRF2000 = 2
+uint32 COORDINATE_SYSTEM_WGS84 = 3
+uint32 COORDINATE_SYSTEM_LOCAL = 4
+uint64 COORDINATE_SYSTEM_UNAVAILABLE = 255
```

Comment:
Why is this one uint64?

Reply:
I don't remember actually. I did this long ago in the Mondeo-repo but it should work with uint32. I'll test it and change